File: uri/1074.c
#include <stdio.h>

int main() {
    int n, i;
    int num[10000];
    scanf("%d", &n);
    for (i=0; i<n; i++) {
        scanf("%d", &num[i]);
    }
    for (i=0; i<n; i++) {
        if (num[i] == 0) printf("NULL\n");
        else if (num[i] > 0 && num[i] % 2 == 0) printf("EVEN POSITIVE\n");
        else if (num[i] > 0 && num[i] % 2 != 0) printf("ODD POSITIVE\n");
        else if (num[i] < 0 && num[i] % 2 == 0) printf("EVEN NEGATIVE\n");
        else if (num[i] < 0 && num[i] % 2 != 0) printf("ODD NEGATIVE\n");
    }
    return 0;
}

File: uri/1074.java
import java.io.*;
import java.util.*;
import java.math.*;

public class Main {

    public static void main (String[] args) throws IOException {

        MyScanner sc = new MyScanner();
        out = new PrintWriter(new BufferedOutputStream(System.out));

        int n = sc.nextInt();

        for (int i = 1; i <= n; i++) {
            int x = sc.nextInt();
            String msg = new String();

            if (x == 0)                     msg = new String("NULL");
            else if (x < 0 && (x % 2 == 0)) msg = new String("EVEN NEGATIVE");
            else if (x < 0 && (x % 2 != 0)) msg = new String("ODD NEGATIVE");
            else if (x > 0 && (x % 2 == 0)) msg = new String("EVEN POSITIVE");
            else if (x > 0 && (x % 2 != 0)) msg = new String("ODD POSITIVE");

            out.println(msg);
        }

        out.close();
    }

    public static PrintWriter out;

    public static class MyScanner {
        BufferedReader br;
        StringTokenizer st;

        public MyScanner() {
            br = new BufferedReader(new InputStreamReader(System.in));
        }

        String next() {
            while (st == null || !st.hasMoreElements()) {
                try {
                    st = new StringTokenizer(br.readLine());
                } catch (IOException e) {
                    e.printStackTrace();
                }
            }
            return st.nextToken();
        }

        int nextInt() {
            return Integer.parseInt(next());
        }

        long nextLong() {
            return Long.parseLong(next());
        }

        double nextDouble() {
            return Double.parseDouble(next());
        }

        String nextLine(){
            String str = "";
            try {
                str = br.readLine();
            } catch (IOException e) {
                e.printStackTrace();
            }
            return str;
        }

    }
}
